src/drivers/block_dev/block_dev.c

```diff
@@ -29,18 +25,10 @@
 
 #include <embox/block_dev.h>
 
-#define MAX_DEV_QUANTITY OPTION_GET(NUMBER,dev_quantity)
+#define MAX_DEV_QUANTITY 16 /* XXX */
```

Comment:
"max_dev_quantity" should be declared in "block_common" Mybuild module, as pools are defined there.
At the same, "max_dev_quantity" are required in "block" module, which depends on "block_common" and which is VFS-dependent.
I have no idea how to use MAX_DEV_QUANTITY with "option_get" in this situation.